apps/frontend/src/modules/projects/components/ThemeCard.tsx

```diff
@@ -26,6 +27,11 @@ export const ThemeCard = ({
         return theme.colorScheme[key];
     };
 
+    const editTheme = () => {
+        useStudioStore.setState({ editTheme: true });
```

Comment:
@kaf-lamed-beyt This is not right. We shouldn't be accessing studio in project module. It breaks dependency flow, and will make it difficult to test this component. Studio should be able to reference Theme but not vice-versa.

Reply:
Since the state is needed in the studio that was why I placed it in the state local to the studio. Should I move it into the project module? Perhaps creating a `states` dir in it?